### PR TITLE
feature: Add gyro configuration options over NT

### LINKED ISSUE #96: Simple Moving Average Smoothing in romi-gyro.ts adds up to a 50ms-100ms.
The code I am referencing is here:
https://github.com/wpilibsuite/wpilib-ws-robot-romi/blob/main/src/robot/romi-gyro.ts

Before continuing I should preface this by saying that I have only used the gyroscope with this smoothing enabled. I am unaware of how noisy it was before this. 

I think it would be nice if this was configurable in the Web Interface, because with the current setup, this moving average adds about 100ms of latency to the gyroscope. The encoders do not have this delay, meaning it's difficult to use the encoders and gyroscope in the same context (differential drive odometer). 

It would be a really cool feature to add to the web interface.

### PR DESCRIPTION
**Pull request by zhiquanyeo**
Resolves #96

### CODE REVIEW

**Comment by jpokornyiii** [comment]
Why did you choose to add additional offset instead of just tweaking the existing values?

File: src/robot/devices/core/lsm6/lsm6.ts [hunk 1]
```diff
@@ -529,13 +549,13 @@ export default class LSM6 {
             if (shouldProcess) {
                 switch (i % 6) {
                     case 0:
-                        tempFrame.gyroX = ((val * gyroScaleFactor) / 1000) - this._gyroOffset.x;
+                        tempFrame.gyroX = ((val * gyroScaleFactor) / 1000) - this._gyroOffset.x + this._gyroAdditionalOffset.x;
```
Comment (jpokornyiii) [inline src/robot/devices/core/lsm6/lsm6.ts:552]: Why add additional offset instead of subtract like original offset from calibration?

Comment (zhiquanyeo) [reply src/robot/devices/core/lsm6/lsm6.ts:552]: It's mainly to allow on-the-fly adjustment in robot code (if necessary) and act in conjunction with the pre-calibrated zero offset (and not replace it)

Comment (zhiquanyeo) [reply src/robot/devices/core/lsm6/lsm6.ts:552]: Could go either way. The zero offsets were all positive, hence the subtraction.

**Pull request merged**